src/components/BatchVoting/DefaultOptions/AbstainOptions.tsx

```diff
@@ -0,0 +1,64 @@
+// Copyright 2019-2025 @polkassembly/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+import React from 'react';
+import { Form, FormInstance } from 'antd';
+import BalanceInput from '~src/ui-components/BalanceInput';
+import BN from 'bn.js';
+import { useTheme } from 'next-themes';
+import styled from 'styled-components';
+
+export enum EFormType {
```

Comment:
exports shouldn't happen from random files, this will create cyclic dependencies, please put it in types file